test/tic_tac_toe_test.exs

```diff
@@ -18,7 +18,11 @@ defmodule TicTacToeTest do
     end
 
     test "(unit test) displays welcome message and board" do
-      assert TicTacToe.start(WriterMock) ===
+      start_supervised!(CommunicatorMock)
+
+      TicTacToe.start(CommunicatorMock, CommandLineFormatter)
```

Comment:
Should we be using a mock in place of `CommandLineFormatter` here?

Reply:
This is a good question and something I was debating as well, Eli and I talked about it and I think we came to the conclusion that a mock wasn't needed here as there is nothing external (like IO) that the formatter interacts with.